src/OrleansServiceBus/Providers/Streams/EventHub/EventHubQueueCache.cs

```diff
@@ -162,7 +228,7 @@ internal class AveragingCachePressureMonitor
         public AveragingCachePressureMonitor(double flowControlThreshold, Logger logger)
         {
             this.flowControlThreshold = flowControlThreshold;
-            this.logger = logger.GetSubLogger("flowcontrol", "-");
+            this.logger = logger.GetSubLogger("flowcontrol-averaging-cache-pressure", "-");
```

Comment:
Suggestion:
As we're moving more towards an asp.net logging model, I suggest using the typename here rather than custom name.
Also, delilimiter argument is optional, and "-" is default, so it should not be needed.
so:
this.logger = logger.GetSubLogger(GetType().Name);